lib/main.dart

```diff
@@ -35,6 +44,41 @@ class _MyAppState extends State<MyApp> {
   @override
   void initState() {
     super.initState();
+
+    final QuickActions quickActions = QuickActions();
+    quickActions.initialize((String shortcutType) {
+
+      if (shortcutType == 'action_one') {
+        _service.sendSms(number);
+      } else {
+      _service.call(number);
+      }
+      setState(() {
```

Comment:
Since we are not showing shortcut value anywhere in UI we can remove this code, but check after removing is it making a difference.